### PR TITLE
updates to the login page

### PR DESCRIPTION
**Pull request by BidemiEnoch**
I updated the login.html, login.js and login.css pages

### CODE REVIEW

File: client/login.html [hunk 1]
```diff
@@ -16,12 +16,17 @@ <h3 class="logo">Entrepre<span>neurship</span></h3>
...
-          <button type="submit" name="Login">Login</button>
-      </form>
+      <div id="container">
+        <div id="header">AUL Forum Login</div>
+        <div id="form-holder">
+          <form id="login-form">
+            <input name="matric-no" type="text" placeholder="Matric Number">
+            <input name="password" type="password" placeholder="Password">
+            <button type="submit">LOG IN</button>
+            <button>SIGN UP</button>
```
Comment (Tekipeps) [inline client/login.html:26]: why is there is a sign up button in login page?

**Review by Tekipeps — COMMENTED** [review]
Good PR

I dont understand why the package.json was removed

File: client/login.html (hunk 1 above)
Comment (BidemiEnoch) [reply client/login.html:26]: standard login forms have a sign up button beneath the login button, the package.json was accidentally removed, dont merge it yet anyways, ill make another updated pull request

Comment (BidemiEnoch) [reply client/login.html:26]: the sign up button would redirect to the register page

Comment (Tekipeps) [reply client/login.html:26]: ok cool.. also add `yarn-error.log` to gitignore

**Pull request closed without merging**